src/server/frappy/frappapi/fixtures/all.json

```diff
@@ -251,7 +251,8 @@
             "create_date": "2022-10-23T04:36:09.503Z",
             "comments": "",
             "final_price": "0.0",
-            "menu_key": 1
+            "menu_key": 1,
```

Comment:
Marking all known static drinks as done so they don't get returned to us when we queue for drinks that are in progress.